site/gatsby-site/cypress.config.js

```diff
@@ -2,7 +2,6 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   video: false,
-  videoUploadOnPasses: false,
```

Comment:
why delete this?

Reply:
Because I see this message on our deployment logs. Nothing to do with the actual `parseNews` problem but I take this opportunity to remove it :)

<img width="835" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/0a5009e4-f8f6-4f47-ab75-75fd06cb17e8">